eve/main.yml

```diff
@@ -150,7 +150,7 @@ stages:
           env:
             ISO_MOUNTPOINT: >
               /srv/scality/metalk8s-%(prop:metalk8s_short_version)s
-          command: tox -e tests-local
+          command: /usr/local/bin/tox -e tests-local
```

Comment:
No, we might install `tox` only for the `eve` user, which would land in `/home/eve/.local/bin`